code/WordpressImporter.php

```diff
@@ -601,9 +601,6 @@ public function parseHTML($content) {
 			else
 				$content .= "<p>$paragraph</p>";
 		}
-		
-		// Split single-line blocks with line-breaks
-		$content = nl2br($content);
```

Comment:
Just some quick food for thought... can you think of any reason why this _should_ be here? Could there be some content that needs to have new lines represented as html breaks?